charger/go-e/api.go

```diff
@@ -78,20 +82,28 @@ func (c *LocalAPI) response(partial string, res interface{}) error {
 }
 
 // Status reads a v1/v2 api response
-func (c *LocalAPI) Status() (Response, error) {
-	if c.v2 {
-		res := new(StatusResponse2)
-		err := c.response("status?filter=alw,car,eto,nrg,wh,trx,cards", &res)
-		return res, err
-	}
+func (c *LocalAPI) Status() (res Response, err error) {
+	if time.Since(c.updated) > c.cache {
+
```

Comment:
leerzeile raus

Reply:
ping

Reply:
Also jetzt wirst Du aber pingelig 😂
Moment

Reply:
Done

Reply:
Yo, is mein Job. Ich muss es ja warten ;)